go/service/chat_local.go

```diff
@@ -537,7 +539,7 @@ func (h *chatLocalHandler) getConversationInfo(ctx context.Context,
 	conversationInfo.TopicType = conversationRemote.Metadata.IdTriple.TopicType
 
 	if len(conversationRemote.MaxMsgs) == 0 {
-		return chat1.ConversationInfoLocal{}, nil,
+		return conversationInfo, maxMessages,
```

Comment:
I just didn't want to return `nil` for `maxMessages`, returning blank would have been fine too.